sdk/src/Core/Amazon.Runtime/Internal/Auth/QueryStringSigner.cs

```diff
@@ -57,6 +57,7 @@ public override void Sign(IRequest request, IClientConfig clientConfig, RequestM
             request.Parameters["SignatureVersion"] = SignatureVersion2;
             request.Parameters["SignatureMethod"] = clientConfig.SignatureMethod.ToString();
             request.Parameters["Timestamp"] = AWSSDKUtils.GetFormattedTimestampISO8601(clientConfig);
+            request.SignedAt = clientConfig.CorrectedUtcNow;
```

Comment:
Seems funny now there is a third way being done to get the corrected UTC time between the AWS4Signer and CloudFrontSigner.

Reply:
I went with this because `AWSSDKUtils.GetFormattedTimestampISO8601` uses `clientConfig.CorrectedUtcNow` which ends up calling  `CorrectClockSkew.GetCorrectedUtcNowForEndpoint`. However, I agree it is a bit confusing and will change it.